cmd/root.go

```diff
@@ -20,3 +23,16 @@ func Command() *cobra.Command {
 
 	return cli
 }
+
+// machineID is a helper to return unique ID
+// of the machine.
+func machineID() (*string, error) {
+	id, err := ioutil.ReadFile("/etc/machine-id")
```

Comment:
I get your point, however we should not forget that `machineID()` is a helper to provide a **default** value for the `--id` flag. In case one wants to use a different ID he can just use `--id my-id` or even `--id $(cat /tmp/my-id-file)`.

If we really want to test this helper, we can still rely on fs abstraction with `afero.FS` but it becomes a bit overkill IMHO. :)